SQL/0000-00-00-schema.sql

```diff
@@ -530,8 +530,9 @@ SET SQL_MODE=@OLD_SQL_MODE;
 
 CREATE TABLE `mri_scan_type` (
   `ID` int(11) unsigned NOT NULL auto_increment,
-  `Scan_type` text NOT NULL,
-  PRIMARY KEY  (`ID`)
+  `Name` VARCHAR(255) NOT NULL,
```

Comment:
```suggestion
  `MriScanTypeName` VARCHAR(255) NOT NULL,
```

I would tend to name the fields in that table as `MriScanTypeID` and `MriScanTypeName`. The first one is so that we can use `USING` in the queries. The second one is because when looking at the code change, if you need to look for something in the code with the `Name` field, it is really not specific. However, if we use `MriScanTypeName`, then you could grep that easily in the code.

Reply:
I personally do not like the `TableField` naming convention as I find it verbose and repetitive (I often use `SELECT t.field FROM table AS t` in my queries).

The `USING` and grep arguments do make sense. However, I feel like those are answers to self-inflicted problems that are due to :
1. How widespread queries are in our codebase (they should ideally be constrained to some database files).
2. How raw SQL is a lot less searchable than ORMs.

Reply:
burn 🙃

Reply:
@driusan

Reply:
Hhhhm, our current SQL guidelines to not say anything about prefixing fields with the table name, so I assume not doing it would be the standard behaviour. Anyway, I think this is an issue that is broader than this PR, and that it should be discussed, and decided, at a LORIS meeting. I am fine with blocking until then.

Reply:
I hate to disagree with both of you (joking, I love it) but I only like the ID to have the full table name (because its used in linking and as a reference) I find the full table name redundant in other fields, if we are looking for a query we would look for the table itself in the join clause

Reply:
> I would tend to name the fields in that table as MriScanTypeID and MriScanTypeName. The first one is so that we can use USING in the queries. The second one is because when looking at the code change, if you need to look for something in the code with the Name field, it is really not specific. However, if we use MriScanTypeName, then you could grep that easily in the code.

After some reflexion I am ok with applying these suggestions. I'll do it if / when I have some time to work back on this PR.